### ISSUE TITLE
Add information about token supertypes to the 'Developer: Inspect Editor Tokens and Scopes' command

### ISSUE THREAD

**Issue by 0dinD** [microsoft/vscode#103109]
Issue Type: <b>Feature Request</b>

This would be really useful for theme authors who want to find out the available supertypes of a token, without looking in the `semanticTokenTypes` contribution point. It could just be another line between `semantic token type` and `modifiers`:
![inspectTokens](https://user-images.githubusercontent.com/52179873/88200196-20e6e000-cc46-11ea-8d02-6c103165a121.png)

**Comment by aeschli** [comment]
The shown example is maybe not the best motivation as function has no super type (nor has member).

But I see that for a custom type `pragmaFunction` we could also add `super types`

To act on this, it would be good to have a real-world example that shows the problem. That way we can find the best solution.
There's already a lot of information in this hover, and while great for experts, is probably already too much for others.

**Comment by 0dinD** [comment]
Yeah, the example might have been bad for showing my point. This feature request relates to #103097 , and just aims to make it easier for theme authors to see what language-specific tokens are supertyped to. Feel free to implement this in another way, what I said above was just an example of what you could do.

An example of where this might be useful is with tokens like `annotation` and `annotationMember` in the `redhat.java` extension. It would also be very useful for all the supertypes I would declare in one of the hypothetical solutions for the problem I discussed in #103097 . 

I also agree that the command has a lot of information as it is, but this is information that I think would be very useful to know for theme authors. One idea for reducing the amount of information is to split the command into one for TextMate scopes and another for semantic tokens. But that's just an idea, I don't know if it's actually going to be helpful.
